engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js

```diff
@@ -82,7 +82,7 @@ angular.module('Bastion.errata').controller('ErrataController',
         });
 
         $scope.goToNextStep = function () {
-            IncrementalUpdate.setBulkErrata(nutupane.getAllSelectedResults());
+            IncrementalUpdate.setBulkErrata(nutupane.getAllSelectedResults('errata_id'));
```

Comment:
I also make this change in #6868.  This and the change I mention above are both part of the fix for http://projects.theforeman.org/issues/20293.